README.md

```diff
@@ -51,6 +51,44 @@ The `init` target (in `Common/default.mk`) copies distribution files from resour
 make help                    # Show all available targets
 ```
 
+The `make help` command can be customized with project-specific information by adding metadata to your `package.json` (Node.js projects) or `composer.json` (PHP projects):
+
+```json
+{
+  "name": "my-project",
+  "description": "A brief description of your project",
+  "homepage": "https://docs.example.com",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/example/my-project.git"
+  },
+  "infra": {
```

Comment:
Note, Composer has a schema and doesn't allow arbitrary blocks in the JSON by default, this needs to be inside `extra`, see
https://getcomposer.org/doc/04-schema.md#extra

Not sure if package.json is the same, make sure.